### PR TITLE
Feature/more units

### PR DESCRIPTION
**Pull request by TrevorVonSeggern**

### CODE REVIEW

File: Common/UnitDefinitions/MassFlow.json [hunk 1]
```diff
@@ -20,6 +20,19 @@
         }
       ]
     },
+    {
+      "SingularName": "GramPerDay",
+      "PluralName": "GramsPerDay",
+      "FromUnitToBaseFunc": "x/86400",
+      "FromBaseToUnitFunc": "x*86400",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Deca", "Hecto", "Kilo", "Mega" ],
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply Common/UnitDefinitions/MassFlow.json]: I know that I need all these prefixes, but I can't be sure if it's absolutely necessary for others.

File: Common/UnitDefinitions/Duration.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "yr", "year", "years" ]
+          "Abbreviations": [ "yr", "year", "Year", "years" ]
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply Common/UnitDefinitions/Duration.json]: Yep, I can create a case-insensitive compare in another pull request.

File: Common/UnitDefinitions/Volume.json [hunk 3]
```diff
@@ -447,6 +447,18 @@
...
+    },
+    {
+      "SingularName": "AcreFoot",
+      "PluralName": "AcreFeet",
+      "FromUnitToBaseFunc": "x/0.000810714",
+      "FromBaseToUnitFunc": "x*0.000810714",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ac" ]
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply Common/UnitDefinitions/Volume.json]: It should be changed to "Ac-ft" and also maybe acre-foot and acre-feet? Good catch btw.

File: Common/UnitDefinitions/VolumeFlow.json [hunk 4]
```diff
@@ -176,6 +212,19 @@
...
+    {
+      "SingularName": "LiterPerDay",
+      "PluralName": "LitersPerDay",
+      "FromUnitToBaseFunc": "x/86400000",
+      "FromBaseToUnitFunc": "x*86400000",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "LPD" ]
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply Common/UnitDefinitions/VolumeFlow.json]: Definitely "l/day" and "L/d" are more common, I'll add them.

File: Common/UnitDefinitions/VolumeFlow.json [hunk 5]
```diff
@@ -124,7 +136,19 @@
...
+      "FromBaseToUnitFunc": "x*113007",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cy/day" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "MillionUsGallonPerDay",
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply Common/UnitDefinitions/VolumeFlow.json]: I think "MillionUsGallonPerDay" is more consistent with the other singular unit names, but I don't feel too strongly either way.

File: Common/UnitDefinitions/VolumeFlow.json [hunk 6]
```diff
@@ -291,6 +389,54 @@
...
+    },
+    {
+      "SingularName": "AcreFootPerHour",
+      "PluralName": "AcreFeetPerHour",
+      "FromUnitToBaseFunc": "x/2.91857",
+      "FromBaseToUnitFunc": "x*2.91857",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "af/m" ]
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply Common/UnitDefinitions/VolumeFlow.json]: Yep

File: UnitsNet.Tests/CustomCode/VolumeFlowTests.cs [hunk 7]
```diff
@@ -91,6 +91,46 @@ public class VolumeFlowTests : VolumeFlowTestsBase
...
+
+        protected override double UkGallonsPerDayInOneCubicMeterPerSecond => 19005304;
+
+        protected override double UkGallonsPerMinuteInOneCubicMeterPerSecond => 13198.2;
+
+        protected override double UkGallonsPerSecondInOneCubicMeterPerSecond => 219.969;
+
+        protected override double UsGallonsPerDayInOneCubicMeterPerSecond => 22824465.227;
+
+        protected override double MegaukGallonsPerSecondInOneCubicMeterPerSecond => 219.969 * 1e-6;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Use scientific notation

**Comment by angularsen** [comment]
Nuget 4.2.0 on the way out
https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.2.0

File: UnitsNet.Tests/CustomCode/VolumeFlowTests.cs [hunk 8]
```diff
@@ -45,9 +45,9 @@ namespace UnitsNet.Tests.CustomCode
 {
     public class VolumeFlowTests : VolumeFlowTestsBase
     {
-        protected override double CubicMetersPerHourInOneCubicMeterPerSecond => 3600.0;
+        protected override double CubicMetersPerHourInOneCubicMeterPerSecond => 3.6e3;
```
[the remark this answers is not in the document]
Comment (TrevorVonSeggern) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Yeah I figured it wasn't required, but once I started making them scientific notation, it was harder for me to stop than to keep going.  :)

[the remark this answers is not in the document]
Comment (angularsen) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Haha 😆

**Pull request merged**